lib/modules/platform/bitbucket/index.ts

```diff
@@ -275,11 +275,15 @@ export async function getPrList(): Promise<Pr[]> {
   logger.debug('getPrList()');
   if (!config.prList) {
     logger.debug('Retrieving PR list');
-    let url = `/2.0/repositories/${config.repository}/pullrequests?`;
-    url += utils.prStates.all.map((state) => 'state=' + state).join('&');
+    const querySearchParams = new URL.URLSearchParams();
+    for (const state of utils.prStates.all) {
+      querySearchParams.append('state', state);
+    }
     if (renovateUserUuid && !config.ignorePrAuthor) {
-      url += `&q=author.uuid="${renovateUserUuid}"`;
+      querySearchParams.append('q', `author.uuid="${renovateUserUuid}"`);
     }
+    const query = querySearchParams.toString();
+    const url = `/2.0/repositories/${config.repository}/pullrequests?${query}`;
```

Comment:
```suggestion
    const url = `/2.0/repositories/${config.repository}/pullrequests?${queryquerySearchParams}`;
```
why not simply this?

Reply:
Because of eslint